uui/components/pickers/PickerInput.tsx

```diff
@@ -95,7 +101,15 @@ export class PickerInput<TItem, TId> extends PickerInputBase<TItem, TId, PickerI
             <IEditableDebouncer
                 value={ targetProps.value }
                 onValueChange={ this.handleTogglerSearchChange }
-                render={ (editableProps) => renderTarget({ ...this.getTogglerMods(), ...targetProps, ...editableProps }) }
+                render={ (editableProps) => renderTarget({
+                    ...this.getTogglerMods(),
+                    ...targetProps,
+                    ...editableProps,
+                    rawProps: {
+                        ...targetProps.rawProps,
+                        'data-testid': 'uui-PickerInput-target',
```

Comment:
Here it should be possible to use this.props.rawProps?.input to pass any custom data-testid, from test itself